core/src/test/com.github.mmorihiro.core/TestMakeNewGeneration.kt

```diff
@@ -3,15 +3,24 @@ package com.github.mmorihiro.core
 import io.kotlintest.specs.WordSpec
 
 
-internal class TestMakeNewGeneration : WordSpec() {
+class TestMakeNewGeneration : WordSpec() {
     init {
+        val sut = MakeNewGeneration()
         "MakeNewGeneration.get" should {
             "return the same size array" {
-                val sut = MakeNewGeneration()
                 val expect = 10
                 val array = Array(expect) { BooleanArray(expect) { false } }
                 sut.get(array).size shouldBe expect
             }
+
+            """make new generation
+            when there are three living cell adjacent to one dead cell""" {
```

Comment:
I almost always agree to use `trimMargin` with multi line string.
Please rewrite it and the code gets better.
Sad to say,
Intellij doesn't have any support for converting ordinary multi line string into the form that uses `trimMargin` in Kotlin even though Intellij has the support for Scala.